pyvista/plotting/plotting.py

```diff
@@ -5041,6 +5041,11 @@ def add_lines(self, lines, color='w', width=5, label=None, name=None):
             If an actor of this name already exists in the rendering window, it
             will be replaced by the new actor.
 
+        connected: bool, default: False
+            Treat ``lines`` as points representing a series of *connected* lines.
+            For example, two connected line segments would be represented as
+            ``np.array([[0, 0, 0], [1, 0, 0], [1, 1, 0]])``.
```

Comment:
For the sake of unambiguity I'd mention that in the `False` case we need an even number of points. This might help drive the message home.

Reply:
I've added a sentence to hopefully make this clearer, but you're right that's it's probably still not super clear for someone that hasn't used this function before

Reply:
I think most of the ambiguity comes from the fact that "lines" actually means "2-length/primitive lines" whereas a polyline can be arbitrarily long. But I'm not sure it's worth trying to pin down the language; the doctest complete with picture should be clear enough. Hmm, or perhaps we should add the same function call but with the new parameter, to show how _that_ looks. What do you think?

Reply:
> perhaps we should add the same function call but with the new parameter, to show how that looks

I like that! It should clear things up